### DISCUSSION TITLE
Virtual-Event-Starter-Kit

### DISCUSSION THREAD

**Discussion by solomonshalom** [vercel/community#502] [Help] [answered]
![image](https://user-images.githubusercontent.com/71135230/158539795-1e480604-ee8a-4947-a1b8-a794ecd1d5b9.png)

I am getting this error, every time I add the DatoCMS and 100MS integration to the virtual-event-starter-kit. Can anyone help me set this project up? I really want to try this project! Thanks

**Comment by leerob** [comment]
The team is looking into this - thank you! Should be resolved soon.

**Comment by gogocharli** [comment, marked as answer]
Hello @solomonshalom, this should now be resolved. Could you give it another try?
